docs/af5-app-from-scratch/antora.yml

```diff
@@ -0,0 +1,14 @@
+name: af5-app-from-scratch
```

Comment:
This becomes the URL. For SEO purposes, it's better to write words fully.

So, I'd suggest something like `axon-framework-5-application-from-scratch`.

I'd love for a name that resembles you also show the event model and stuff, but can't think of anything, so let's go with from scratch for now

Reply:
Good idea!
I have another propositions. Which one do you prefer? Also @smcvb :
- `axon-framework-5-app-getting-started-from-eventmodeling-to-code`
- `axon-framework-5-getting-started`
- `axon-framework-5-from-eventmodeling-to-code`

Reply:
Agreed